jfrog_platform/assets/dashboards/jfrog_platform_overview.json

```diff
@@ -0,0 +1,901 @@
+{
+    "title": "Jfrog Dashboard",
+    "description": "",

```

Comment:
Can we include a brief description in the dashboard that links to the docs page?